envoy/admin/v2/admin.proto

```diff
@@ -0,0 +1,31 @@
+syntax = "proto3";
```

Comment:
Can we move this to `config_dump.proto`? I think we should have a discrete proto file for each endpoint in the admin package.

Reply:
Yeah, agreed, was just using a catch-all to punt on naming. Will do.